cores/esp8266/Esp.h

```diff
@@ -22,6 +22,7 @@
 #define ESP_H
 
 #include <Arduino.h>
+#include "umm_malloc/umm_malloc_cfg.h"
```

Comment:
> LD  /home/runner/work/Arduino/Arduino/tests/host/bin//home/runner/work/Arduino/Arduino/libraries/ESP8266WiFi/examples/WiFiClient/WiFiClient
/usr/bin/ld: /home/runner/work/Arduino/Arduino/tests/host/bin/fullcore.a(WString.cpp.o): in function `String::changeBuffer(unsigned int)':
/home/runner/work/Arduino/Arduino/cores/esp8266/WString.cpp:234: undefined reference to `heap_pvPortRealloc'

Seems to be because of this include?
Arduino.h also has specific clause for this cfg header wrapped in `#ifdef DEBUG_ESP_OOM ... #endif`, something wrong in the include order?